source/_components/nest.markdown

```diff
@@ -65,7 +65,15 @@ Configuration variables:
 
 Currently there is a single `nest.set_mode` service available to switch between
 "away" and "home" modes. This service requires a `home_mode` param and has an
-optional `structure` param.
+optional `structure` param. While setting "away" mode, an estimated arrival time
+can also be set with `eta`, `eta_window`, and `trip_id` parameters.
+
+- **home_mode** (*Required*): `home` or `away`
+- **structure**: Structure(s). Default apply to all structures connected with Homa Assistant.
+- **eta**: Estimated Arrival Time from now.
+- **eta_window**: Optional ETA window. Default is 1 minute.
```

Comment:
Change line to: `**eta_window** (*Optional*): ETA window in minutes (default: 1).`

Reply:
Not exactly right, since the param type has been changed to time_period